newswires/client/src/urlState.ts

```diff
@@ -51,7 +51,7 @@ export function urlToConfig(location: {
 	} else if (page.startsWith('item/') && page.split('/').length === 2) {
 		return { view: 'item', itemId: page.split('/')[1], query };
 	} else {
-		console.warn(`Page not found: "${page}", so using defaultConfig`);
+		console.log(`Page not found: "${page}", so using defaultConfig`);
```

Comment:
Having this as a `warn` was making the logs quite noisy when running tests. Given that we're not storing client-side logs in production, there doesn't seem much benefit to keeping this as a `warn`, as opposed to a `log`.